consul/README.md

```diff
@@ -134,23 +134,23 @@ Optionally, you can configure Consul to also send data to the Agent through [Dog
            match_type: "regex"
            name: "consul.http.request"
            tags:
-             http_method: "$1"
+             method: "$1"
              path: "$2"
          - match: 'consul\.raft\.replication\.appendEntries\.logs\.([0-9a-f-]+)'
            match_type: "regex"
            name: "consul.raft.replication.appendEntries.logs"
            tags:
-             consul_node_id: "$1"
+             peer_id: "$1"
          - match: 'consul\.raft\.replication\.appendEntries\.rpc\.([0-9a-f-]+)'
            match_type: "regex"
            name: "consul.raft.replication.appendEntries.rpc"
            tags:
-             consul_node_id: "$1"
+             peer_id: "$1"
          - match: 'consul\.raft\.replication\.heartbeat\.([0-9a-f-]+)'
            match_type: "regex"
            name: "consul.raft.replication.heartbeat"
            tags:
-             consul_node_id: "$1"
+             peer_id: "$1"
```

Comment:
Are these changes related to Consul 1.9.0 support? The tag names here are "arbitrary" (users can choose whichever they'd like), right?

Reply:
Ah I see! — it's in the PR description:

> I updated the README instructions for DogStatsD user to have the same tags as the prometheus metrics

_Maybe_ in general it's better to keep these unrelated changes in separate PRs? I won't nitpick and we can merge this, but just an idea for the future.

Reply:
My thought was that this readme change is kind of related to Consul 1.9.0, but a separate PR may have been better indeed